packages/workflow/src/Expression.ts

```diff
@@ -268,7 +281,11 @@ export class Expression {
 		const returnValue = this.renderExpression(extendedExpression, data);
 		if (typeof returnValue === 'function') {
 			if (returnValue.name === '$') throw new Error('invalid syntax');
-			throw new Error('This is a function. Please add ()');
+
+			if (returnValue.name === 'DateTime')
```

Comment:
Should we check this for some other types? Is `DateTime` the only one that can be rendered here?

Reply:
What do you mean? These are all functions.

Reply:
Was thinking about this `returnValue.name === 'DateTime'`, is only `DateTime` requiring this check here?

Reply:
`{{ DateTime }}` resolves to a function and the default error message does not apply because here we need to access its static methods. I cannot think of any other case like this but please let me know if I'm missing anything.

Reply:
Not sure, just checking.